src/util/state-management.js

```diff
@@ -82,7 +82,13 @@ function overwriteArrays (objValue, srcValue) {
 	}
 }
 
-export function buildStatefulComponent(baseComponent, opts) {
+export function buildHybridComponent(baseComponent, opts) {
+
+	if (baseComponent._isLucidHybridComponent && process.NODE_ENV !== 'production') {
```

Comment:
and, actually, about that `production` check, i didn't find anywhere else where we're doing that. Is that how we want to do it?